sample/src/androidTest/kotlin/io/github/kakaocup/sample/screen/TestActivityScreen.kt

```diff
@@ -19,7 +19,7 @@ open class TestActivityScreen : Screen<TestActivityScreen>() {
     val singleTypeRecycler: KButton = KButton { withId(R.id.single_type_recycler) }
     val autoComplete: KButton = KButton { withId(R.id.auto_complete) }
     val snackbarButton: KButton = KButton { withId(R.id.snackbar_button) }
-    val toolbar: KToolbar = KToolbar { withId(R.id.action_bar) }
+    val toolbar: KToolbar = KToolbar { withId(com.google.android.material.R.id.action_bar) }
```

Comment:
Yes. Because ToolBar has been added implicitly via activity configuration (not in XML). Technically it no longer presenting in module R file, that is why it was removed and replaced by global Google constant.

I guess that change related to the gradle feature which disables by default automatic modules resources merge (what actually makes sense).